### PR TITLE
Audit managing-configuration-groups-for-cloud-databases-with-the-trove-command-line-tool.md: INFODEV-1363

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

File: content/cloud-databases/managing-configuration-groups-for-cloud-databases-with-the-trove-command-line-tool.md [hunk 1]
```diff
@@ -298,30 +296,30 @@ The command returns the properties of the configuration group.
...
     trove configuration-update <config_id> <value_pairs> [--name <name>] [--description <description>]
 
 For example, you might start with an existing group with two values set,
 `character_set_server` and `max_connections`, as shown in the following
-table.
+table:
 
 | Key                    | Value  |
 |------------------------|--------|
 | `character_set_server` | `utf8` |
```
Comment (ayersek64) [inline content/cloud-databases/managing-configuration-groups-for-cloud-databases-with-the-trove-command-line-tool.md]: missing max_connections

**Comment by RackerKate** [comment]
@cyrichardson Another for your review please.

**Comment by cyrichardson** [comment]
On it, @RackerKate!

**Comment by cyrichardson** [comment]
@RackerKate Suggestions here: 
[managing-configuration-groups-for-cloud-databases-with-the-trove-command-line-tool_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1925873/managing-configuration-groups-for-cloud-databases-with-the-trove-command-line-tool_cyrEdits.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-39835ccb1b/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d11f0ccfcb/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7696df5c05/how-to/).

**Pull request merged**